### DISCUSSION TITLE
Svelte Content Script UI?

### DISCUSSION THREAD

**Discussion by cowboycodr** [wxt-dev/wxt#530] [Q&A] [answered]
I'm trying to spin up a quick chrome extension using Svelte.

However, all of the tutorials I've found so far just so basic popups, and sometimes sidebars. Yet I want   to utilize svelte components inside of content scripts.

I've found that this framework supports using React in content scripts but couldn't find an example for Svelte.

I'm sure it's possible, but I would really appreciate a bit of help in doing so.

Thank you in advance!

**Comment by aklinker1** [comment, marked as answer]
https://wxt.dev/guide/content-script-ui.html#shadow-root

The demo code block has a Svelte example.

![Screenshot_20240306-184415.png](https://github.com/wxt-dev/wxt/assets/10101283/2d828c90-c994-4045-abf8-2fea87ba95e9)

**Reply by cowboycodr** [reply]
You're a legend bro. Thanks.
